# Zwart Lab Tools
A list of tools routinely used by the Zwart lab for genomics analysis and visualization

## Differential analysis <a name="differential analysis"></a>

For the basics differential binding and gene expression

**DESeq2 for differential gene expression of RNA-seq experiments:** [https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html]

**DiffBind for differential binding analysis of ChIP-seq experiments:** [https://bioconductor.org/packages/release/bioc/vignettes/DiffBind/inst/doc/DiffBind.pdf]


## Basic analysis of ChIP-seq data <a name="Basic analysis of ChIP-seq data"></a>

**UCSC Galaxy Cistrome Server for running motif and genomic regions enrichment:** [http://cistrome.org/ap/root]

**CistromeDB Toolkit Giggle analysis (hg38 only!!!!):** [http://dbtoolkit.cistrome.org/]



## Visualization of ChIP-seq data <a name="Visualization of ChIP-seq data"></a>

**DeepTools for heamaps and profiles (linux server):** [https://deeptools.readthedocs.io/en/develop/]

**EaSeq for heatmaps and profiles (Windows):** [https://easeq.net/]

## Visualization <a name="Visualization"></a>

**ggplot2 in R:** [https://ggplot2.tidyverse.org/, https://r4ds.had.co.nz/data-visualisation.html, http://www.sthda.com/english/wiki/ggplot2-box-plot-quick-start-guide-r-software-and-data-visualization]

**ggpubr in R for statistics in your plot:** [https://rpkgs.datanovia.com/ggpubr/]


## Getting help <a name="getting help"></a>

Some useful pointers for where to look for help online. 

**Biostars -- Sometimes your bioinformatic question has already been asked:** [https://www.biostars.org/]

**(2019-04) Deep learning: new computational modelling techniques for genomics** [[Nature Reviews Genetics paper](https://www.nature.com/articles/s41576-019-0122-6)]


